From bd7264ac2f189c5bfa7776604976c655dc5e1647 Mon Sep 17 00:00:00 2001
From: Brent Stapleton <brent.stapleton@ettus.com>
Date: Wed, 6 Mar 2019 09:55:55 -0800
Subject: [PATCH 1/4] python: change CMake variable for library ext

Changing the CMake variable used in determining the extension of the
Python API library for better cross-platform support.

Specifically, `CMAKE_SHARED_MODULE_SUFFIX` corresponds to .so for *nix
and MacOS, which is the extension Python expects.
---
 host/python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/python/CMakeLists.txt b/host/python/CMakeLists.txt
index d0a6e2204..a19391d83 100644
--- a/host/python/CMakeLists.txt
+++ b/host/python/CMakeLists.txt
@@ -33,7 +33,7 @@ target_link_libraries(pyuhd ${BOOST_PYTHON_LIBRARY} ${Boost_LIBRARIES} ${PYTHON_
 if(WIN32)
     set(PYUHD_LIBRARY_NAME libpyuhd.pyd)
 else()
-    set(PYUHD_LIBRARY_NAME libpyuhd${CMAKE_SHARED_LIBRARY_SUFFIX})
+    set(PYUHD_LIBRARY_NAME libpyuhd${CMAKE_SHARED_MODULE_SUFFIX})
 endif(WIN32)
 
 add_custom_command(TARGET pyuhd
-- 
2.20.1

